account_payment_show_invoice/models/account_payment.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountPayment(models.Model):
+    _inherit = 'account.payment'
+
+    @api.multi
```

Comment:
@lreficent I change from api.one -> api.multi. In the past, I heard that we should always use api.multi since api.one have performance issue.
But I also see many code using api.one, not sure the performance was fixed. Please suggest.